pom.xml

```diff
@@ -0,0 +1,495 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><!--
+ ~ Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ ~
+ ~ WSO2 Inc. licenses this file to you under the Apache License,
+ ~ Version 2.0 (the "License"); you may not use this file except
+ ~ in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~    http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing,
+ ~ software distributed under the License is distributed on an
+ ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ ~ KIND, either express or implied.  See the License for the
+ ~ specific language governing permissions and limitations
+ ~ under the License.
+ --><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.wso2</groupId>
+        <artifactId>wso2</artifactId>
+        <version>1.4</version>
+    </parent>
+    <groupId>org.wso2.carbon.extension.identity.authenticator.outbound.voiceotp</groupId>
+    <artifactId>identity-outbound-auth-voice-otp</artifactId>
+    <version>1.0.0.0</version>
+    <packaging>pom</packaging>
+    <name>WSO2 Carbon Extension - Voice OTP Pom</name>
+    <url>http://wso2.org</url>
+    <profiles>
+        <profile>
+            <id>default</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <modules>
+                <module>component/authenticator</module>
+                <module>feature</module>
+            </modules>
+        </profile>
+    </profiles>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.wso2.carbon.identity.framework</groupId>
+                <artifactId>org.wso2.carbon.idp.mgt</artifactId>
+                <version>5.25.92</version>
```

Comment:
Rather than hard coding the component's version suggest to add a property section in the parent pom file and mentioned the version as a version range. (If we selected a specific version there might be issues when IS version gets bump to a upper version. Hence without having such version ranges we may need to re write another connector to the bump version) Check other occurrences.

Refer : https://github.com/wso2-extensions/identity-oauth2-grant-rest/blob/main/pom.xml#L518-L588